<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vendas_ui</class>
 <widget class="QWidget" name="vendas_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QMenuBar" name="menuBar">
     <widget class="QMenu" name="menuTrocar_Visualiza_o">
      <property name="title">
       <string>Trocar Visualização</string>
      </property>
      <addaction name="actionLotes_Vendas"/>
      <addaction name="actionClientes_Vendas"/>
      <addaction name="actionVendedores_Vendas"/>
      <addaction name="actionVendas_Vendas"/>
     </widget>
     <addaction name="menuTrocar_Visualiza_o"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="Menu_Title">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>32</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>VENDAS</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="APP" stretch="1,6">
         <item>
          <layout class="QHBoxLayout" name="FORM" stretch="2,3,3,3,0,0,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="data_layout">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="data_label">
               <property name="text">
                <string>DATA</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="data_dateEdit">
               <property name="date">
                <date>
                 <year>2023</year>
                 <month>11</month>
                 <day>10</day>
                </date>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="lote_layout">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="lote_label">
               <property name="text">
                <string>LOTE</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="lote_list">
               <property name="tabKeyNavigation">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="vendedor_layout">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="vendedor_label">
               <property name="text">
                <string>VENDEDOR</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="vendedor_list">
               <property name="tabKeyNavigation">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="cliente_layout">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="cliente_label">
               <property name="text">
                <string>CLIENTE</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="cliente_list">
               <property name="tabKeyNavigation">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="btns_layout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="add_btn">
               <property name="text">
                <string>ADICIONAR</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="rm_btn">
               <property name="text">
                <string>REMOVER</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="edit_btn">
               <property name="text">
                <string>EDITAR</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="ListWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionLotes_Vendas">
   <property name="text">
    <string>Lotes</string>
   </property>
  </action>
  <action name="actionClientes_Vendas">
   <property name="text">
    <string>Clientes</string>
   </property>
  </action>
  <action name="actionVendas_Vendas">
   <property name="text">
    <string>Vendas</string>
   </property>
  </action>
  <action name="actionVendedores_Vendas">
   <property name="text">
    <string>Vendedores</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
